nix/pkg/default.nix

```diff
@@ -67,8 +78,7 @@ assert lib.assertMsg (stdenv.isLinux || !gamemodeSupport) "gamemodeSupport is on
         their associated options with a simple interface.
       '';
       platforms = with platforms; linux ++ darwin;
-      changelog = "https://github.com/PrismLauncher/PrismLauncher/releases/tag/${version}";
```

Comment:
why this? :O

Reply:
the current link is broken and i don't really know of a good way to represent the changelog given a commit. i feel like it would be a a bit weird to just link to the commit diff. my other thought was sending the comparison of the nearest tag to the latest commit, but getting that tag and/or hard coding it in the expression would be more problems than it's worth imo